### PR TITLE
Fix Clazy v1.12 errors in main

### PR DESCRIPTION
**Pull request by JoergAtGithub**
This is the Follow-Up of https://github.com/mixxxdj/mixxx/pull/13769 addressing additional Clazy v1.12 errors in main

### CODE REVIEW

File: src/test/controllerrenderingengine_test.cpp [hunk 1]
```diff
@@ -34,7 +34,8 @@ class MockRenderingEngine : public ControllerRenderingEngine {
 };
 
 TEST_F(ControllerRenderingEngineTest, createValidRendererWithSupportedTypes) {
-    for (auto pixelFormat : supportedPixelFormat()) {
+    const auto& supportePixelFormats = supportedPixelFormat();
+    for (const auto& pixelFormat : supportePixelFormats) {
```
Comment (Swiftb0y) [inline src/test/controllerrenderingengine_test.cpp:38]:
```suggestion
    const auto& supportedPixelFormats = supportedPixelFormat();
    for (const auto& pixelFormat : supportedPixelFormats) {
```

**Review by Swiftb0y — COMMENTED** [review]
LGTM otherwise.

**Comment by Swiftb0y** [comment]
LGTM. squash?

**Review by cr7pt0gr4ph7 — APPROVED** [review]
Looks good :+1:

**Review by Swiftb0y — APPROVED** [review]

**Pull request merged**